apps/onboarding/src/gateway/gateway.service.ts

```diff
@@ -59,6 +59,12 @@ export class GatewayService implements OnModuleInit {
       })
     )
 
+    results.forEach(result => {
+      if (result.ok === false) {
+        this.logger.warn(result.error)
+      }
+    })
+
     return results.every(result => result.ok)
```

Comment:
Optional: we can roll this value calculation into the loop above to prevent looping through twice.